src/pages/dashboard/charts/_AreaChart.js

```diff
@@ -60,7 +60,12 @@ const AreaChart = ({
   const strokeColor = '#b2bbbf'
 
   return (
-    <svg viewBox={`0 0 ${width} ${height}`} style={{ overflow: 'visible' }}>
+    <svg
+      width={width}
+      height={height}
```

Comment:
This completely breaks the layout of the small area charts below. I had to switch this to using `viewBox` in order to scale the small area charts responsively across multiple breakpoints.

I think fix this instead by constraining the `AreaChart` in a container with the desired dimensions. It might even be preferable and feasible to put the two charts in a row together, like the ones in the "Differences between this data and the CDC data" section higher up in the dashboard.

Reply:
<img width="1920" alt="Screen Shot 2020-04-12 at 1 28 00 PM" src="https://user-images.githubusercontent.com/63169602/79075535-757ed480-7cc1-11ea-9384-fe2cda3270db.png">